### ISSUE TITLE
Recommended settings for style training

### ISSUE THREAD

**Issue by asukii314** [TheLastBen/fast-stable-diffusion#1427]
Hey there - apologies if this is a basic question, couldn't seem to find a clear answer. I've got ~180ish images in the same style that I'd like to train a model around. Other than ticking the style training box after >1k steps, I'm wondering what other settings I should use:

- How many of those ~180 images should I actually train with?
- Assuming I don't use all of them, is it worth uploading the unused ones as concept images, or should I save that as a backup plan in case I get bad results otherwise?
- How many training steps should I start with, on both the unet and text encoder? (and I suppose the text encoder concept training part too, if using that)
- Is there any benefit to changing the learning rate from the default for style training?
- Not necessary at all, but for bonus points: could I safely train in one or more faces at the same time as training the style? (And if so, how many different faces do you think would be feasible, + are there any other settings I'd need to adjust to compensate?)

Many thanks in advance!

**Comment by TheLastBen** [comment]
Use 10 images and keep the default settings, don't tick the style_training box, also try the v2, it's very good with styles.

**Comment by 2blackbar** [comment]
what if i must use about 50 images to properly represent all items drawn in a certain style, should i train text encoder more or unet for more steps than text enc?

**Comment by TheLastBen** [comment]
train the unet more

**Comment by 2blackbar** [comment]
Id add a line to the cel - best settings for training person likeness with 20 pics and best settings for training style with 100 pics
Not sure if i should train text enc just foir like 200 steps and leave it at 0 later when training unet only to get better likeness., also learning rates, i dont want to overfit.
Theres no math formula in there like it was previosly (about 100steps per 1 image) so im pretty lost there

**Comment by TheLastBen** [comment]
training is subjective, some settings may work for some images and won't for others, so you'll have to do trial & error
